native/golang/main.go

```diff
@@ -75,9 +75,16 @@ func main() {
 	}
 
 	// Print ready message to stdout as JSON
+	var data *map[string]any = nil
+	if checksums := utils.LoadChecksums(); checksums != nil {
+		data = &map[string]any{
+			"checksums": checksums,
+		}
+	}
 	readyMsg := t.StatusMessage{
 		Status:  "ready",
 		Message: "zowed is ready to accept input",
+		Data:    data,
```

Comment:
Can we avoid the use of a pointer here?

```suggestion
	data := make(map[string]any)
	if checksums := utils.LoadChecksums(); checksums != nil {
		data["checksums"] = checksums
	}
	readyMsg := t.StatusMessage{
		Status:  "ready",
		Message: "zowed is ready to accept input",
		Data:    data,
```

I guess the difference here is that data would be an empty object on the response with this approach (`{}` vs. `undefined`), so maybe that isn't the preferred behavior 😋 

Alternatively, we could assign the value of `utils.LoadChecksums` directly to a key in the map, and when the checksums are absent we know based on the serialization of `nil`:

```suggestion
	data := make(map[string]any)
	data["checksums"] = utils.LoadChecksums()
	readyMsg := t.StatusMessage{
		Status:  "ready",
		Message: "zowed is ready to accept input",
		Data:    data,
```

Reply:
Good suggestion, addressed in d69370048d4e26e3761bebee3cdfe1fddb7cd2c1